styx-scheduler-service/src/test/java/com/spotify/styx/flyte/FlyteAdminClientRunnerTest.java

```diff
@@ -126,7 +126,7 @@ public void testThrowsCreateExecutionExceptionForOtherCode() {
 
     assertThrows(
         FlyteRunner.CreateExecutionException.class,
-        () -> flyteRunner.createExecution("exec", FLYTE_EXEC_CONF));
+        () -> flyteRunner.createExecution(null, "exec", FLYTE_EXEC_CONF));
```

Comment:
when do we expect null runstate? If never, I would say lets change the tests in this file and add validation on FlyteRunner